packages/admin-ui/src/Radio/Radio.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import * as RadioGroupPrimitive from "@radix-ui/react-radio-group";
+import { cn, makeDecoratable } from "~/utils";
+
+/**
+ * RadioItem
+ */
+type RadioProps = React.ComponentPropsWithoutRef<typeof RadioGroupPrimitive.Item> & {
+    label: string | React.ReactNode;
+};
+
+const DecoratableRadio = React.forwardRef<
+    React.ElementRef<typeof RadioGroupPrimitive.Item>,
+    RadioProps
+>(({ className, label, id, ...props }, ref) => {
```

Comment:
Will the users need to pass `id` all the time in order to have label also clickable / act as if a user clicked on the radio button? 🤔 

Can we generate this maybe in cases the user did not pass the ID?